src/shared/os-util.c

```diff
@@ -117,3 +117,26 @@ int load_os_release_pairs(const char *root, char ***ret) {
 
         return load_env_file_pairs(f, p, ret);
 }
+
+int load_os_release_pairs_container_prefix_env(const char *root, char ***ret) {
+        _cleanup_strv_free_ char **os_release_pairs = NULL;
+        char **p, **q, *line;
+        int r;
+
+        r = load_os_release_pairs(root, &os_release_pairs);
+        if (r < 0)
+                return r;
+
+        if (strv_isempty(os_release_pairs))
+                return -1;
+
+        STRV_FOREACH_PAIR(p, q, os_release_pairs) {
+                if (asprintf(&line, "container_host_%s=%s", *p, *q) < 0)
+                        return -ENOMEM;
+                r = strv_consume (ret, line);
```

Comment:
no space bweteen function name and `(` please

Reply:
Sorry, keep making this mistake as another project I spend a lot of time on mandates it :-)